src/App.tsx

```diff
@@ -95,6 +109,7 @@ const App: React.FC = () => {
 
     // Tree structure
     try {
+      const IS_FILE = "blob";
       const res = await fetch(
         `https://api.github.com/repos/${owner}/${repo}/commits/master`
```

Comment:
URL Should be abstracted as well
CONST GITHUB API URL PREFIX
CONST GITHUB API COMMITS ON MASTER SUFFIX etc